cli/src/upload.rs

```diff
@@ -113,6 +113,37 @@ pub struct UploadArgs {
     pub allow_empty_test_results: bool,
 }
 
+impl UploadArgs {
+    pub fn new(
+        token: String,
+        org_url_slug: String,
+        junit_paths: Vec<String>,
+        repo_root: String,
+    ) -> Self {
+        Self {
+            junit_paths,
+            #[cfg(target_os = "macos")]
+            xcresult_path: None,
+            org_url_slug,
+            token,
+            bazel_bep_path: None,
+            repo_root: Some(repo_root),
+            repo_url: None,
+            repo_head_sha: None,
+            repo_head_branch: None,
+            repo_head_commit_epoch: None,
+            tags: Vec::new(),
+            print_files: false,
+            dry_run: false,
+            team: None,
+            // TODO - how are code owners handled?
+            codeowners_path: None,
```

Comment:
What are you missing here primarily?

Reply:
Main question is how are we grabbing code owners when running this way. I don't know what the shape of it is going to look like.